src/sdp.c

```diff
@@ -367,6 +367,9 @@ int janus_sdp_process_remote(void *ice_handle, janus_sdp *remote_sdp, gboolean r
 		/* Is simulcasting enabled, using rid? (we need to check this before parsing SSRCs) */
 		tempA = m->attributes;
 		medium->rids_hml = rids_hml;
+		medium->rid[0] = NULL;
+		medium->rid[1] = NULL;
+		medium->rid[2] = NULL;
```

Comment:
While I check the previous code, this is definitely not a proper fix yet, though. You're just setting those variables to `NULL` without freeing them, which means this will cause a memory leak.